.travis.yml

```diff
@@ -20,9 +20,24 @@ install:
   # PR to add macOS support to Pester
   # https://github.com/pester/Pester/pull/925
   - git clone --branch PSCore_compatibility_clean --depth 1 https://github.com/it-praktyk/Pester.git $HOME/Pester
+  # Download script helper to convert Pester coverage into coverage JSON format for codecov
+  - pwsh -c "{
+      $uri = 'https://raw.githubusercontent.com/PowerShell/DscResource.Tests/491688867dc53894b92ca53520a18d145deb7760/DscResource.CodeCoverage/CodeCovIo.psm1';
```

Comment:
It's not a published module?

Reply:
Unfortunately no. I imagine Pester to receive built-in support for this soon, but this works, is a pinned revision so it can't break and is quick enough to download.
Was mentioned here: https://github.com/pester/Pester/issues/212#issuecomment-280202296